.github/workflows/release.yml

```diff
@@ -45,21 +54,30 @@ jobs:
         with:
           arguments: --filter=-mypy
 
+      - name: Build Go Bindings
+        if: "${{ github.event.inputs.tag != '' }}"
+        env:
+          RESIM_VERSION: ${{ github.event.inputs.tag }}
+        run: |
+          git config --global user.name 'Michael Bauer'
+          git config --global user.email 'michael@resim.ai'
```

Comment:
We could add a service account to do this as well I think.

Reply:
Yeah makes sense, but this is probably fine for now since all our usual releases are tagged by a ReSim employee anyways. I do think we have a service account already 🤔 Deployment bot: https://github.com/resim-ai/deployment/commit/a8e0454364ad07e654f227c627ab7f360446be6e
so maybe Ben could help with that.

Maybe could add a backlog task if it doesn't make sense to swap it over now? These fun little easter eggs always explode when someone leaves the company 😆

Reply:
Trying to copy https://github.com/resim-ai/resim-ts-client/blob/main/.github/workflows/codegen.yml#L29-L30 for this.